### DISCUSSION TITLE
Create branches on your name by the end of today and commit and push the changes to your branch.

### DISCUSSION THREAD

**Discussion by enigma037** [Orchid-DOTNET/BscCSIT-2023#7] [General]
After you clone the repo
=> git clone paste url
=> Make sure you are inside the folder BscCSIT-2023 ( cd BscCSIT-2023)
=> git checkout -b YourFirstNameLastName
=> Add your name folder inside Section B/ or Section A folder and add Assignment1.cs file
=> git add .
=> git commit -m "Added assignment 1"
=> git push origin YourFirstNameLastName

**Comment by enigma037** [comment]
@mgmr99  Make sure you are inside the folder BscCSIT-2023 ( cd BscCSIT-2023)
This might be the reason it was not allowed you to create branch.

**Comment by mgmr99** [comment]
Throwing Error!!

**Reply by enigma037** [reply]
Can you paste the screenshot here? @mgmr99

**Comment by enigma037** [comment]
@Narotsit-Karki @sregmi02 **Add your FirstnameLastname folder inside Section B/ or Section A folder and add Assignment1.cs file**
